firestore/pom.xml

```
@@ -37,6 +37,9 @@
   <properties>
     <maven.compiler.target>1.8</maven.compiler.target>
     <maven.compiler.source>1.8</maven.compiler.source>
+    <!-- Define a default empty value property which can be overridden on the command line if
+         necessary by passing `-Dfirestore.project.id="some-id"` when running maven exec -->
+    <firestore.project.id/>
```

Rather than an empty cell, how about a default value that the user can change in the pom (if they prefer) 
```suggestion
    <firestore.project.id>YOUR-PROJECT-ID</firestore.project.id>
```

---

By having it defined as an empty value we can also support the scenario it is run with `mvn exec:exec` and have the project id inherited from the service credentials. From what I understand we're trying to nudge users in that direction, but we're not there yet across the board with docs.

I think having the empty tag is just as easy for a user to update if they're interested without any impact to them if they don't. If we don't want the short tag, I can expand that.

---

That makes sense to me. 

FWIW, I think we are moving both away from command line samples and using system properties to configure.

---

yeah, the system property is a bit annoying when dealing with the maven runners.